fastn-core/src/commands/serve.rs

```diff
@@ -489,6 +491,10 @@ async fn handle_static_route(
     }
 }
 
+async fn handle_migrations(config: &fastn_core::Config) -> fastn_core::Result<()> {
+    Ok(fastn_core::migrations::migrate(config).await?)
+}
```

Comment:
Not a useful function, call `fastn_core::migrations::migrate(config)` directly from the `serve` function.

Reply:
done